version.go

```diff
@@ -21,4 +21,4 @@
 
 package cadence
 
-const Version = "v0.31.3"
+const Version = "v0.31.5"
```

Comment:
So the next tag going to be `v0.31.6`?

Reply:
Yes, if we release another patch after this. This PR is just making public what was already released, so no need to bump the version here one more time.